twstock/codes.py

```diff
@@ -24,7 +24,7 @@
 
 def read_csv(path, types):
     global codes, twse, tpex
-    with open(path, newline='') as csvfile:
+    with open(path, newline='', encoding='utf_8_sig') as csvfile:
```

Comment:
Could you help to add the comment to explain why using `utf_8_sig`? It seam `utf_8_sig` will handle BOM, right?